src/app/parcels/ActionBar/Actions.tsx

```diff
@@ -149,6 +161,7 @@ const ActionsButton: React.FC<ActionsButtonProps> = ({
 
 interface Props {
     selected: number[];
+    setSelected: React.Dispatch<React.SetStateAction<number[]>>;
     data: ParcelsTableRow[];
```

Comment:
Shall we rename this to `parcels` or something instead of a very generic `data`?

Reply:
Have done (and similar occurances)